ml/src/test/scala/ai/h2o/sparkling/ml/features/H2OWord2VecTestSuite.scala

```diff
@@ -33,53 +37,75 @@ class H2OWord2VecTestSuite extends FunSuite with Matchers with SharedH2OTestCont
 
   override def createSparkSession(): SparkSession = sparkSession("local[*]")
 
-  private lazy val dataset = spark.read
+  private lazy val craigslistJobTitles = spark.read
     .option("header", "true")
     .option("inferSchema", "true")
     .csv(TestUtils.locate("smalldata/craigslistJobTitles.csv"))
+    .select("jobtitle")
 
-  private lazy val Array(trainingDataset, testingDataset) = dataset.randomSplit(Array(0.9, 0.1), 42)
-
-  def getPipeline(): Pipeline = {
+  def similarJobTitlesPipeline(): Pipeline = {
     val tokenizer = new RegexTokenizer()
       .setInputCol("jobtitle")
+      .setGaps(false)
+      .setToLowercase(true)
+      .setPattern("[a-zA-Z]+")
       .setMinTokenLength(2)
 
+    val jobListingsStopWords =
+      Array("hiring", "job", "fulltime", "parttime", "opening", "part", "time", "full", "needed", "seeking", "seek")
+
     val stopWordsRemover = new StopWordsRemover()
+      .setStopWords(loadDefaultStopWords("english") ++ jobListingsStopWords)
       .setInputCol(tokenizer.getOutputCol)
 
     val w2v = new H2OWord2Vec()
       .setSentSampleRate(0)
       .setEpochs(10)
+      .setVecSize(101)
+      .setOutputCol("Word2VecOutput")
       .setInputCol(stopWordsRemover.getOutputCol)
 
-    val gbm = new H2OGBM()
-      .setLabelCol("category")
-      .setFeaturesCols(w2v.getOutputCol)
+    val kMeans = new H2OKMeans()
+      .setK(100)
+      .setSeed(42)
+      .setFeaturesCol(w2v.getOutputCol())
+      .setPredictionCol("cluster")
 
-    new Pipeline().setStages(Array(tokenizer, stopWordsRemover, w2v, gbm))
+    new Pipeline().setStages(Array(tokenizer, stopWordsRemover, w2v, kMeans))
   }
 
-  test("H2OWord2Vec Pipeline serialization and deserialization") {
-    getPipeline().write.overwrite().save("ml/build/w2v_pipeline")
-    val loadedPipeline = Pipeline.load("ml/build/w2v_pipeline")
-    val model = loadedPipeline.fit(trainingDataset)
-    val expected = model.transform(testingDataset)
-
-    model.write.overwrite().save("ml/build/w2v_pipeline_model")
```

Comment:
IMHO, it's good to check that persistence of the model doesn't affect the model behavior.

Reply:
not sure why I removed it, probably by mistake, it's back now, thanks